script/openqa-bootstrap

```diff
@@ -112,7 +112,8 @@ if command -v $setup; then
 else
     curl -s https://raw.githubusercontent.com/os-autoinst/openQA/master/script/configure-web-proxy | bash -ex -s -- "$proxy_args"
 fi
-sed -i -e 's/#*.*method.*=.*$/method = Fake/' /etc/openqa/openqa.ini
+echo -e "[auth]\nmethod = fake" > /etc/openqa/openqa.ini.d/01-enable-fake-auth.ini
+
```

Comment:
```suggestion
```

Reply:
That's actually what our style check also prefers :)

https://app.circleci.com/pipelines/github/os-autoinst/openQA/16559/workflows/48b49a0a-fc23-4b7f-bd6f-557f166d23e1/jobs/160393